docs/sftt/sftt-api-spec.md

```diff
@@ -2204,6 +2204,42 @@ If the `activity_id` specified is not associated with an existing activity.
 
 If the calling user is not an admin or the user listed on the activity.
 
+### Edit Stewardship Activity (Admin and Author Only)
+
+`POST api/v1/protected/sites/edit_stewardship/:activity_id`
+
+Edits an existing stewardship activity. Indicate `True` if the activity (watered, mulched, etc.) was completed, else `False`.  At least one activity must be `True`.
```

Comment:
Can we add this to make it clearer that `date` is the date that the activity was performed?: `date` is the day on which the activity was performed, which can be in the past if the activity was performed in the past.